misc/jenkins/functional_test_and_build_bundle/run.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 git submodule update --init
-bash misc/jenkins/functional_test_and_build_bundle/build_current_bundle.sh
```

Comment:
huh? just like that - just do not build for HW CI? let's extract into separate PR?

Reply:
HW CI doesn't run that script; I don't know what it's for.

Reply:
Not trying to attach just want to confirm: "i am not sure what this script does but I have confidence to make interesting changes"?

Reply:
I am fine doing https://github.com/rusefi/rusefi/commit/d3c12dc6f53c77918de227bb7cb997ce2e4faabe and https://github.com/rusefi/rusefi/commit/33a26d05f7e209f4894557fa16dc9225aae04d18